src/librustdoc/html/static/rustdoc.css

```diff
@@ -542,11 +542,13 @@ h4 > code, h3 > code, .invisible > code {
 }
 
 .content .stability::before {
-	content: '˪';
-	font-size: 30px;
+	content: '⤶';
+	transform:rotate(90deg);
+	font-size: 20px;
 	position: absolute;
-	top: -9px;
-	left: -13px;
+	top: -6px;
+	left: -15px;
+	color: #999;
```

Comment:
Everything color-linked should be put inside light.css and dark.css files (even if it's the same in both files.